src/clj/ladders/db.clj

```diff
@@ -0,0 +1,74 @@
+(ns ladders.db
+  (:require [datomic.api :as d]
+            [ladders.config :as conf]))
+
+(def uri (:datomic-uri conf/config))
+
+(defonce conn (atom nil))
+
+(defn connect []
+  (reset! conn (d/connect uri)))
+
+(defn transact [data]
+  (if-not (nil? @conn)
+    (d/transact @conn data)))
+
```

Comment:
The schema definition is pretty rough on the eyes.  It'd be nice to just chat through it at a whiteboard for 2 minutes.